locales/pt-BR/translation.json

```
@@ -0,0 +1,93 @@
+{
+  "$Language": {
+    "en": "English",
+    "pt-BR": "Português (Brasil)"
+  },
+  "$Preferences": {
+    "title": "Time to Leave - Preferências",
+    "userPreferences": "Preferences de usuário",
+    "workingDays": "Dias uteis",
+    "sun": "Dom",
+    "mon": "Seg",
+    "tue": "Ter",
+    "wed": "Qua",
+    "thu": "Qui",
+    "fri": "Sex",
+    "sat": "Sab",
+    "hideNonWorkingDay": "Ocultar finais de semana (Visão de mês)",
```

This is misleading. Technically if I select my working days to be only "Sunday+Saturday", this would hide everything but the weekend
```suggestion
    "hideNonWorkingDay": "Ocultar dias não úteis (Visão de mês)",
```